schemas/gen.py

```diff
@@ -299,22 +301,48 @@ def go_getters(struct: str, schema: Schema, spec: List[FieldSpec]) -> List[str]:
     for field, type, tag in spec:
         defaulted_type, default, required = get_defaulted_type(schema, tag, type)
 
+        if not field.startswith("Raw"):
+            raise AssertionError(
+                f'{struct} has field {field} which doesn\'t start with "Raw"; all fields should '
+                'start with "Raw" and the getter will be the primary API for accessing those '
+                "values.  When the field is a pointer-type with a non-nil default type, the getter "
+                "will be a non-pointer (automatic dereferencing) for use after WithDefaults() is "
+                "used to fill the default values."
+            )
+
+        getter = field[len("Raw") :]
+
         if defaulted_type == type:
-            lines.append(f"func ({x} {struct}) Get{field}() {type} {{")
+            # Getter for nonpointer field.
+            lines.append(f"func ({x} {struct}) {getter}() {type} {{")
             lines.append(f"\treturn {x}.{field}")
             lines.append("}")
             lines.append("")
+
+            # Setter for nonpointer field.
+            lines.append(f"func ({x} {struct}) Set{getter}(val {type}) {{")
+            lines.append(f"\t{x}.{field} = val")
+            lines.append("}")
+            lines.append("")
+
         else:
-            lines.append(f"func ({x} {struct}) Get{field}() {defaulted_type} {{")
+            # Getter for pointer field.
+            lines.append(f"func ({x} {struct}) {getter}() {defaulted_type} {{")
             lines.append(f"\tif {x}.{field} == nil {{")
             lines.append(
-                f'\t\tpanic("You must call WithDefaults on {struct} before .Get{field}")'
+                f'\t\tpanic("You must call WithDefaults on {struct} before .{field}")'
             )
             lines.append("\t}")
             lines.append(f"\treturn *{x}.{field}")
             lines.append("}")
             lines.append("")
 
+            # Setter for pointer field.
+            lines.append(f"func ({x} {struct}) Set{getter}(val {defaulted_type}) {{")
```

Comment:
same comment here about pointer receivers. potentially couldn't hurt to make generated code lintable, if this is an issue I think it would've caught it.

Reply:
I can look into this.  I don't have any interest in e.g. teaching gen.py how to reflow long lines, but maybe I disable only certain linters.

Reply:
Eh, it's fine for now. I'm not sure anything would've caught it actually.